docs/galaxy_releases.rst

```diff
@@ -0,0 +1,218 @@
+
+  # The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+  # content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+  # underscores or numbers and cannot contain consecutive underscores
+  namespace: namespace_name
+
+  # The name of the collection. Has the same character restrictions as 'namespace'
+  name: collection_name
+
+  # The version of the collection. Must be compatible with semantic versioning
+  version: "{{ tag }}"
+
+  # The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+  readme: README.md
+
+  # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+  # @nicks:irc/im.site#channel'
+  authors:
+    - "Author1"
+    - "Author2 (https://author2.example.com)"
+    - "Author3 <author3@example.com>"
+
+
+  ### OPTIONAL but strongly recommended
+
+  # A short summary description of the collection
+  description: Description of what this collection has
+
+  # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+  # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+   license:
+     - GPL-2.0-or-later
+
+  # The path to the license file for the collection. This path is relative to the root of the collection. This key is
+  # mutually exclusive with 'license'
+  # license_file: ''
+
+  # A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+  # requirements as 'namespace' and 'name'
+  tags: []
+
+  # Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+  # collection label 'namespace.name'. The value is a version range
+  # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+  # range specifiers can be set and are separated by ','
+  dependencies: {}
+
+  # The URL of the originating SCM repository
+  repository: https://github.com/my_org/my_collection
+
+  # The URL to any online docs
+  documentation: https://github.com/my_org/my_collection
+
+  # The URL to the homepage of the collection/project
+  homepage: https://github.com/my_org/my_collection
+
+  # The URL to the collection issue tracker
+  issues: https://github.com/my_org/my_collection/issues
+
+
+Configure GitHub Workflow
+-------------------------
+
+To be able to execute the Ansible playbook we need to configure the GitHub
+Actions Workflow. To do this we will need to create the
+:code:`.github/workflows/` folder if it does not already exist. Then create a
+new file :code:`release.yml` and it should look something like this.
+
+.. code-block:: yaml
+
+  name: "release"
+  on:
+  release:
+    types:
+      - created
+  jobs:
+  release:
+    runs-on: ubuntu-18.04
```

Comment:
Probably better to use
```suggestion
    runs-on: ubuntu-latest
```

Reply:
So what do you do when 20.04 is the latest and doesn't have the expected packages or support?

Reply:
Assuming latest is best is never a good way to go, at least from any operations perspective because latest is unpredictable in most including docker as a tag of latest can and may get moved to a release your code isn't tested on.

Reply:
there was a problem when GH was moving fast and dropped support for previous macOS versions, for example. and it was PITA to upgrade the hardcoded thing everywhere. I'm just trying to avoid the same. Besides, it's not like you're using some extraordinary software here that may suddenly disappear.